lms-types/src/lib.rs

```diff
@@ -53,6 +56,16 @@ impl LmsAlgorithmType {
     pub const LmsSha256N24H25: Self = Self::new(14);
 }
 
+#[cfg(feature = "std")]
+impl<'de> Deserialize<'de> for LmsAlgorithmType {
```

Comment:
nit: can you include a comment for why Deserialize needs to be manually implemented instead of derived?